keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -486,7 +491,7 @@ def train_step(self, data):
         if sample_weight is not None:
             raise ValueError("`sample_weight` is currently not supported.")
         gt_boxes = y["boxes"]
-        gt_classes = y["classes"]
+        gt_classes = tf.expand_dims(y["classes"], axis=-1)
```

Comment:
Can you add a TODO for me to remove this hack?

Reply:
Also, given this is backward breaking, we should have a safety-check here: if the input doesn't have it, expand the dimension, otherwise keep as-is

Reply:
Added a safety check raising a value error.  Lets require proper inputs instead of trying to massage inputs to be properly formatted.

Reply:
added todo